### PR TITLE
[job status change notif] handle the alerts in alert-manager

### PR DESCRIPTION
**Pull request by suiguoxin**
Part 3 in [Work Items](https://github.com/microsoft/pai/issues/5337#issuecomment-839551679)

### CODE REVIEW

File: src/alert-manager/deploy/alert-templates/job-status-change/html.ejs [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+    PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
+
+<html xmlns="http://www.w3.org/1999/xhtml" xmlns="http://www.w3.org/1999/xhtml"
+    style="font-family: 'Helvetica Neue', Helvetica, Arial, sans-serif; box-sizing: border-box; font-size: 14px; margin: 0;">
+
+<body>
+    Dear OpenPAI user:
+    </br>
+    </br>
+    Some of your jobs in OpenPAI cluster <b><%= cluster_id %></b> have changed status.
```
Comment (hzy46) [inline src/alert-manager/deploy/alert-templates/job-status-change/html.ejs:11]: Status of your job(s) has changed in OpenPAI cluster?

File: src/alert-manager/deploy/alert-manager-configmap.yaml.template [hunk 2]
```diff
@@ -73,6 +77,13 @@ data:
       - url: 'http://localhost:{{ cluster_cfg["alert-manager"]["alert-handler"]["port"] }}/alert-handler/send-email-to-admin/?template=cluster-usage'
         send_resolved: false
       {% endif %}
+    
+    - name: pai-job-status-change
+      webhook_configs:
+      {% if 'email-user' in cluster_cfg["alert-manager"]["actions-available"] %}
+      - url: 'http://localhost:{{ cluster_cfg["alert-manager"]["alert-handler"]["port"] }}/alert-handler/send-email-to-admin/?template=job-status-change'
```
Comment (hzy46) [inline src/alert-manager/deploy/alert-manager-configmap.yaml.template:84]: Does `send-email-to-admin` just send mails to administrator? I think mails should be sent to corresponding users.

File: src/alert-manager/deploy/alert-templates/job-status-change/html.ejs (hunk 1 above)
Comment (suiguoxin) [reply src/alert-manager/deploy/alert-templates/job-status-change/html.ejs]: > Status of your job(s) has changed in OpenPAI cluster?

Fixed

File: src/alert-manager/deploy/alert-manager-configmap.yaml.template (hunk 2 above)
Comment (suiguoxin) [reply src/alert-manager/deploy/alert-manager-configmap.yaml.template]: > Does `send-email-to-admin` just send mails to administrator? I think mails should be sent to corresponding users.

Yes, it should be `send-email-email-to-user`

**Review by hzy46 — APPROVED** [review]

**Pull request merged**